### ISSUE TITLE
Style the sections using the design template

### ISSUE THREAD

**Issue by WebDevJL** [Puzzlout/TrekToursFlyer#13]
@damirius I am separating this from issue #9. This issue is to implement the design of the template (uploaded in #9) into the views you have created.

The vertical menu be another issue as well.

Let me know if you have questions.

**Comment by WebDevJL** [comment]
@damirius 

So far so good. Now we need to use more of the draft design I sent:
- can you try to make each section at least take the whole viewport size or fullscreen?
- try to style the section to look like the sections of the draft design (See Google Drive folder: Assets/Design).

Thanks.

**Comment by damirius** [comment]
@WebDevJL 

We have one similar issue #37, which is also about styling sections. We must agree on theme colors for our website and so on. I mean I know we have that sample design, but do we want to just copy their design together with colors and everything or we are using it just for example/inspiration?
Can you tell me if there is a logo ready for the website? If it is we can see which colors are used there so we use the same colors for our headers,links and so on

**Comment by WebDevJL** [comment]
@damirius use the same white over black and black over white every two sections.

We have no logo yet since the name is not final yet. So forget the logo for now.

Stéphane really likes the black and white pictures and the overall feel of the design so let's go for that! @gsouvik Same thing about your styling tasks.

Thanks.

**Comment by damirius** [comment]
@WebDevJL 

New stuff is on feature_product_page. I've opted to use some of the assets you shared with us. I chose 3 images and made grayscale version of them to use as fixed background like on that example design you sent us. It would be great if we could get some bigger images, so that we can work with that, but in the meantime check this out if you think it's OK. To be honest just black/white sections feel really empty, so I opted to put those backgrounds there. Tell me if you like it or if you want to change it back
